hack/docker/mysql-helper/docker-entrypoint.sh

```diff
@@ -27,6 +27,11 @@ storage_class = ${GCS_STORAGE_CLASS:-"MULTI_REGIONAL"}
 type = http
 url = ${HTTP_URL}
 
+[azure]
+type = azure blob storage
+account = ${RCLONE_AZUREBLOB_ACCOUNT)
+key = $(RCLONE_AZUREBLOB_KEY)
```

Comment:
Use curly brackets instead of round brackets. I think endpoint should be configurable too:
```
endpoint = ...
```